docs/csharp/how-to/modify-string-contents.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "How to: Modify String Contents (C# Guide)"
+ms.date: 02/26/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "strings [C#], modifying"
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+
+# How to: Modify String Contents (C# Programming Guide)
+
+This article demonstrates several techniques to produce a `string` by modifying an existing `string`. All the techniques demonstrated return the result of the modifications as a `string` object. To clearly demonstrate this, the examples all store the result in a new variable. You can then examine both the original `string` and the `string` resulting from the modification when you run each example. 
+
+There are several techniques demonstrated in this article. You can replace existing text. Other methods search for patterns and replace matching text with other text. You can also treat a string as a sequence of characters. You can also use convenience methods that remove whitespace. YOu should choose the techniques that most closely matches your scenario.
+
+## Replace text
+
+The following code demonstrates this *immutable* property of strings. It creates a new string by replacing existing text with a substitute.
+
+[!code-csharp-interactive[replace creates a new string](../../../samples/snippets/csharp/how-to/strings/ModifyStrings.cs#1)]  
+
+You can see in the preceding example that the original string, `source`, is not modified. The <xref:System.String.Replace%2A?displayProperty=nameWithType> method creates a new `string` containing the modifications.
+
+The <xref:System.String.Replace%2A> method can replace either strings or single characters. In both cases, every occurrence of the sought text is replaced.  The following example replaces all ' ' characters with '_':
+
+[!code-csharp-interactive[replace characters](../../../samples/snippets/csharp/how-to/strings/ModifyStrings.cs#2)]  
+
+The source string is unchanged, and a new string is returned with the replacement.
```

Comment:
I'm not sure why everything from the code label on is italicized. Is '_' interpreted as \*\?

Reply:
I think it's the `_`. The italics does not show up on the review site.